File: Aluno.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace caHeranca
{
    class Aluno : Pessoa
    {
        private String curso;

        public Aluno() : base()
            {
               curso = " ";
            }

        public Aluno(String _nome, int  _idade, String  _curso) : 
            base(_nome, _idade)
        {
            curso = _curso;
            
        }

        public void imprimeDados()
        {
            Console.WriteLine("Dados do Aluno:");
            Console.WriteLine("Nome = " + nome);
            Console.WriteLine("Idade = " + idade);
            Console.WriteLine("Curso = " + curso);
        }
    }
}

File: Heranca/Aluno.py
from Pessoa import Pessoa

class Aluno(Pessoa):
    def __init__(self, nome = '', idade = 0, curso = ''):
        Pessoa.__init__(self, nome, idade)
        self.curso = curso


    def imprimir(self):
        print("Dados do Aluno:")
        print("Nome = " + str(self.nome))
        print("Idade = " + str(self.idade))
        print("Curso = " + str(self.curso))
